message-center.js

```diff
@@ -40,48 +58,48 @@ MessageCenterModule.
           messageObject.message = options.html ? $sce.trustAsHtml(message) : message;
           messageObject.html = !!options.html;
           if (angular.isDefined(options.timeout)) {
-            messageObject.timer = $timeout(function () {
+            messageObject.timer = $timeout(function() {
               messageObject.close();
             }, options.timeout);
           }
           this.mcMessages.push(messageObject);
           return messageObject;
         },
-        remove: function (message) {
+        remove: function(message) {
           var index = this.mcMessages.indexOf(message);
           this.mcMessages.splice(index, 1);
         },
-        reset: function () {
+        reset: function() {
           this.mcMessages = [];
         },
-        removeShown: function () {
+        removeShown: function() {
           for (var index = this.mcMessages.length - 1; index >= 0; index--) {
             if (this.mcMessages[index].status == this.status.shown) {
               this.remove(this.mcMessages[index]);
             }
           }
         },
-        markShown: function () {
+        markShown: function() {
           for (var index = this.mcMessages.length - 1; index >= 0; index--) {
             if (!this.mcMessages[index].processed) {
               if (this.mcMessages[index].status == this.status.unseen) {
                 this.mcMessages[index].status = this.status.shown;
                 this.mcMessages[index].processed = true;
-              }
-              else if (this.mcMessages[index].status == this.status.next) {
+              } else if (this.mcMessages[index].status == this.status.next) {
                 this.mcMessages[index].status = this.status.unseen;
               }
             }
           }
         },
-        flush: function () {
+        flush: function() {
           $rootScope.mcMessages = this.mcMessages;
         }
       };
     }
   ]);
 MessageCenterModule.
-  directive('mcMessages', ['$rootScope', 'messageCenterService', function ($rootScope, messageCenterService) {
+directive('mcMessages', ['$rootScope', 'messageCenterService',
+  function($rootScope, messageCenterService) {
```

Comment:
I'm OK having this in a new line, but let's add a space after `function` to keep consistency.